benefit/data/sheets/CaseEventToFields/CaseEventsToFields-writeAdjournmentNotice.json

```diff
@@ -0,0 +1,332 @@
+[
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCasePanelMember1",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "OPTIONAL",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 3,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "panelMemberNames",
+    "PageLabel": "Panel members",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCasePanelMember2",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "OPTIONAL",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 3,
+    "PageFieldDisplayOrder": 2,
+    "PageID": "panelMemberNames",
+    "PageLabel": "Panel members",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCasePanelMember3",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "OPTIONAL",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 3,
+    "PageFieldDisplayOrder": 3,
+    "PageID": "panelMemberNames",
+    "PageLabel": "Panel members",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseTypeOfHearing",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 4,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "typeOfHearing",
+    "PageLabel": "Type of hearing",
+    "PageShowCondition": "adjournCaseGenerateNotice = \"Yes\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseCanCaseBeListedRightAway",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 5,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "canCaseBeListedRightAway",
+    "PageLabel": "Can case be listed right away?",
+    "PageShowCondition": "adjournCaseGenerateNotice = \"Yes\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseAreDirectionsBeingMadeToParties",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 6,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "areDirectionsBeingMadeToParties",
+    "PageLabel": "Are you making directions to the parties?",
+    "PageShowCondition": "adjournCaseGenerateNotice = \"No\" OR adjournCaseCanCaseBeListedRightAway = \"No\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CallBackURLMidEvent": "${CCD_DEF_TRIBUNALS_API_URL}/ccdMidEventAdjournDirectionDueDate",
```

Comment:
can we follow the same naming convention as they've done for the other midEvent
${CCD_DEF_TRIBUNALS_API_URL}/ccdMidEventAdjournCasePopulateVenueDropdown